examples/host_commands.h

```diff
@@ -581,4 +581,15 @@ struct ec_response_persistent_panic_info {
   int32_t persistent_panic_record_version;
 } __attribute__((packed));
 
+// This command allows callers to push initial measurements into PCR0. This
+// command will fail if the TPM has already been initialized, or if the
```

Comment:
nit:
```suggestion
// command will fail if the TPM has already been started up, or if the
```

Init and startup are different things in TPM fw